plugins/GSdx/GSSetting.cpp

```diff
@@ -122,6 +122,13 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_TC_DEPTH:
 			return "Disable the support of Depth buffer in the texture cache.\n"
 				"It can help to increase speed but it will likely create various glitches.";
+		case IDC_8BITCPUFBHACK:
```

Comment:
A better label would be `IDC_8BIT_CPU_FBHACK`

It's a bit of an eyesore in its' current state.